artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/broker/AMQPSessionCallback.java

```diff
@@ -405,7 +405,11 @@ public void ack(Transaction transaction, Object brokerConsumer, Message message)
    public void cancel(Object brokerConsumer, Message message, boolean updateCounts) throws Exception {
       OperationContext oldContext = recoverContext();
       try {
-         ((ServerConsumer) brokerConsumer).individualCancel(message.getMessageID(), updateCounts);
+         try {
+            ((ServerConsumer) brokerConsumer).individualCancel(message.getMessageID(), updateCounts);
+         } catch (IllegalStateException e) {
+            //Ignore this silently, can occur if already cancelled.
```

Comment:
nit-pick: can you log.debug here instead?

Although I could ammend it myself during merging

Reply:
sure will add

Reply:
added - just pushed